docs/requirements.txt

```diff
@@ -4,3 +4,4 @@ myst-parser
 numpydoc
 nbsphinx
 nbsphinx_link
+calibr8
```

Comment:
```suggestion
```
It looks like we're installing from the current directory: https://github.com/JuBiotech/calibr8/blob/02728c20cd6f7f0d7aa9c18eb5eb406f8b471792/.readthedocs.yml#L19